CollapseLauncher/Classes/GenericJSONContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Hi3Helper.EncTool.Parser.AssetMetadata;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+#nullable enable
+namespace CollapseLauncher
+{
+    [JsonSourceGenerationOptions(IncludeFields = false, GenerationMode = JsonSourceGenerationMode.Metadata, IgnoreReadOnlyFields = true)]
+    [JsonSerializable(typeof(int[]))]
+    [JsonSerializable(typeof(AudioPCKType[]))]
+    [JsonSerializable(typeof(Dictionary<string, Dictionary<string, string>>))]
+    internal sealed partial class GenericJsonContext : JsonSerializerContext;
```

Comment:
Might wanna explain what "Generic" in this context means (in code comment), especially when we attach AudioPCKType there (might be better to separate it to its own context?)

Reply:
as per "generic" in this context, meant to be all struct types or "basic", yet built-in collection type (like: Dictionary, List), nor anything which the type doesn't have any category. I don't think it's necessary to separate the context for the items mentioned above.